runtests.py

```diff
@@ -0,0 +1,9 @@
+import sys
```

Comment:
If we go this route will we want to add a shebang. Also a good idea to make it an executable if it isn't already.